fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/services/ContainerServiceImpl.java

```diff
@@ -61,10 +63,22 @@ public Container findOrCreate(final Session session, final String path) {
 
             if (node.isNew()) {
                 initializeNewObjectProperties(node);
+
+                final Node parent = node.getDepth() > 0 ? node.getParent() : null;
+
+                if (parent != null) {
+                    touch(parent);
+                    touchLdpMembershipResource(node);
+                }
+            }
+
+            if (node.isNew()) {
+                touch(node);
             }
 
             return new ContainerImpl(node);
         } catch (final RepositoryException e) {
+            LOGGER.info("Error: {}", e);
```

Comment:
I removed that line in the last commit. Doing so also makes the code consistent with the existing BinaryService::findOrCreate method. Plus, because it rethrows the exception as a RepositoryRuntimeException, it seems like the additional logging is superfluous. Or do you think otherwise? If it goes back in, are you suggesting it be at DEBUG level, containing the whole stacktrace? and should the corresponding BinaryService::findOrCreate function have a similar logging? I'm also beginning to wonder if this is out of scope for this ticket.